docs/trace/extending-the-sdk/README.md

```diff
@@ -256,6 +256,8 @@ Custom processors can be implemented to cover more scenarios:
   and `OnShutdown`.
 * `OnStart` and `OnEnd` should be thread safe, and should not block or take long
   time, since they will be called on critical code path.
+* Processors should use `Activity.TagObjects` collection instead of
```

Comment:
Minor - maybe link to the .NET official doc?

Reply:
That is a good idea. I will update the PR.

Reply:
I think, as a best practice, we should encourage users to use `Activity.EnumerateTagObjects` instead. That's more efficient than simply enumerating over `Activity.TagObjects`.

Reply:
Updated